### ISSUE TITLE
Drop compatibility with bodhi1

### ISSUE THREAD

**Issue by sebwoj** [fedora-infra/bodhi#2947]
I found many code comments where bodhi1 is mentioned, like:
https://github.com/fedora-infra/bodhi/blob/b64d82ce0fff8284164348c6f1d280191f908e7f/bodhi/server/models.py#L3588

Since Bodhi 4.0 is on the way, can we drop bodhi1 compatibility?

**Comment by bowlofeggs** [comment]
Yeah I think we can drop bodhi1 compatibility with Bodhi 4. Thanks!
